tsl/test/isolation/expected/cagg_concurrent_refresh.out

```diff
@@ -1,4 +1,4 @@
-Parsed test spec with 11 sessions
+Parsed test spec with 12 sessions
```

Comment:
It would be good to see concurrency tests with different isolation levels, e.g., REPEATABLE READ. I am wondering what happens if one process generates the "queue" entries while others that should read them are running concurrently in REPEATABLE READ. Will they see the entries with ranges they should refresh?

Reply:
Let's add such kind of isolation test after the release. I think we're good to merge this PR as is.